compiler/luci/pass/src/QuantizeBias.cpp

```diff
@@ -215,6 +217,15 @@ CircleConst *QuantizeBias::quantized_bias(CircleNode *input, const CircleNode *w
       throw std::runtime_error("Unsupported quantization type.");
     }
 
+    auto qp = weight->quantparam();
```

Comment:
I think your intention is to check qparam of bias?

Reply:
Yes

Reply:
Then you'll need to update `weight` to `new_bias`